File: site/sdks/AecClient.java
// KhulnaSoft Engineering Knowledge OS SDK (generated). Do not hand-edit.
package io.khulnasoft;

public final class AecClient {
    private static final String BASE = "https://khulnasoft.github.io/api";

    public static String resources() {
        return BASE + "/resources";
    }

    public static String resource(String slug) {
        return BASE + "/resources/{slug}".replace("{slug}", slug);
    }

    public static String twin(String slug) {
        return BASE + "/resources/{slug}/twin".replace("{slug}", slug);
    }

    public static String context(String slug) {
        return BASE + "/resources/{slug}/context".replace("{slug}", slug);
    }

    public static String recommendations(String slug) {
        return BASE + "/resources/{slug}/recommendations".replace("{slug}", slug);
    }

    public static String graph() {
        return BASE + "/graph";
    }

    public static String impact() {
        return BASE + "/impact";
    }

    public static String events() {
        return BASE + "/events";
    }

    public static String analytics() {
        return BASE + "/analytics";
    }

    public static String release() {
        return BASE + "/release";
    }

}

File: site/sdks/AecClient.cs
// KhulnaSoft Engineering Knowledge OS SDK (generated). Do not hand-edit.
namespace KhulnaSoft;

public static class AecClient
{
    private const string Base = "https://khulnasoft.github.io/api";

    public static string Resources() {
        return Base + "/resources";
    }

    public static string Resource(string slug) {
        return Base + "/resources/{slug}".Replace("{slug}", slug);
    }

    public static string Twin(string slug) {
        return Base + "/resources/{slug}/twin".Replace("{slug}", slug);
    }

    public static string Context(string slug) {
        return Base + "/resources/{slug}/context".Replace("{slug}", slug);
    }

    public static string Recommendations(string slug) {
        return Base + "/resources/{slug}/recommendations".Replace("{slug}", slug);
    }

    public static string Graph() {
        return Base + "/graph";
    }

    public static string Impact() {
        return Base + "/impact";
    }

    public static string Events() {
        return Base + "/events";
    }

    public static string Analytics() {
        return Base + "/analytics";
    }

    public static string Release() {
        return Base + "/release";
    }

}
